src/lib/disabledAccounts.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const _ = require('lodash')
+const Account = require('../models/account').Account
+const UnprocessableEntityError =
+require('five-bells-shared/errors/unprocessable-entity-error')
+
+function * validateNoDisabledAccounts (transaction, transfer) {
+  const accounts = _.groupBy(transfer.debits.concat(transfer.credits), function (creditOrDebit) {
```

Comment:
Again, this was just for consistency. See <a href="https://github.com/interledger/five-bells-ledger/blob/master/src/lib/accountBalances.js#L27">here</a>. But you're probably right that it makes sense to change the style as we go. I'll update the PR.